### ISSUE TITLE
Mod download don't start at all

### ISSUE THREAD

**Issue by brunoarchie** [Nexus-Mods/web-issues#487]
## Bug/error being reported:
I'm unable to download two mods (SkyUI and BVFE). I've tryed through NMM and manually but the download simply just don't start at all. Many other mods are working fine, however. 

## Steps to reproduce:
Download the mods in question through NMM and manually.

## Expected result:
Download the mods via NMM or manually.

## Actual result:
The download don't start at all.

URL: https://www.nexusmods.com/skyrim/mods/3863 and https://www.nexusmods.com/skyrim/mods/38829

**Comment by ousnius** [comment]
I don't know about NMM, but in the browser, did you actually click the second "Download" button in the download info/requirements window that pops up? Download is working for me.

**Comment by adamelders** [comment]
I have the same issue. I can't seem to download any mods. I was able to download mods exactly 10 days ago (1/2/2018). When I try to download a mod from either the Files tab of a mod page, or by the Download button on a mod page, it takes me to a "Your download should automatically begin within a few seconds." page, then nothing happens.

The following errors are logged in Chrome's developer console:

> GET https://www.nexusmods.com/contents/Scripts/BBoC-1515678912.js net::ERR_TOO_MANY_REDIRECTS DownloadPopUp:31
> 
> Uncaught ReferenceError: $ is not defined
>     at DownloadPopUp?id=6379&game_id=1303&source=FileExpander:95 DownloadPopUp?id=6379&game_id=1303&source=FileExpander:95

This is happening on the same computer, with the same internet, same browser, etc. I am using Google Chrome version 63.0.3239.132 (Official Build) (64-bit).

Here are some of the mods I am trying to download:

https://www.nexusmods.com/stardewvalley/mods/1100
https://www.nexusmods.com/darksouls/mods/1372

If you need any more information or debugging help, please let me know.

**Comment by adamelders** [comment]
Ok, for some reason, downloads are now working again. After I switched to another computer, but same browser, browser extensions, same NexusMods account, etc., I was able to download the mods. I switched back to the first computer that was having the trouble, and downloads are working fine again.

The difference was that previously, when I clicked on a download button, it would open a separate page, even though my settings were set to open in a pop up. I also tried toggling this option before, and it still wasn't making any difference (I don't think my changes to that setting were being changed at all).

So, TL;DR, try switching to another computer to download the mod, then switch back if needed. Perhaps logging out and back in to your NexusMods account might also work.
